src/olympia/editors/templates/editors/queue.html

```diff
@@ -162,13 +162,7 @@
             {% for value in row %}
               <td>{{ value|xssafe }}</td>
             {% endfor %}
-            {% if table.show_version_notes %}
-            <td>
-              <div class="addon-version-notes">
-                <a href="#" class="ed-sprite-notes app-icon" title="{{ _('Version Notes / Notes for Reviewer') }}">&nbsp;</a>
-              </div>
-            </td>
-            {% endif %}
+            <td>&nbsp;</td>
```

Comment:
the unlisted all list already skips that column - hence the `{% if...` and why I added the property.  If that doesn't break the layout why is it needed here?